lib/mix/lib/mix/tasks/local.hex.ex

```diff
@@ -75,4 +85,22 @@ defmodule Mix.Tasks.Local.Hex do
         :erlang.raise(kind, reason, stacktrace)
     end
   end
+
+  defp get_matching_version do
+    Mix.Utils.read_path!(@hex_list_url)
+    |> parse_csv
+    |> calculate_version(System.version)
+  end
+
+  defp parse_csv(body) do
+    last_line = :binary.split(body, "\n", [:global, :trim]) |> List.last
+    [_hex|elixirs] = :binary.split(last_line, ",", [:global, :trim])
+    elixirs
```

Comment:
@jw2013 I think it reduces the understanding of the code because ` [_hex|elixirs] = ...` documents better why we skip the head.